README.md

```diff
@@ -61,22 +61,15 @@ npm install wc3maptranslator
 ```
 
 ## Usage
-```js
-var Translator = require('wc3maptranslator'),
-    mapJson = { // Refer to "Specification"
-        units: [...],
-        doodads: [...],
-        ...
-    };
-
-// Using individual translators, we may convert JSON
-// representation to generate a WC3 .w3x map file.
-// See the Wiki for more information.
-
-var unitResult = new Translator.Units.jsonToWar(mapJson.units);
+```ts
+import { Translator } from 'wc3maptranslator';
+const unitResult = new Translator().Units.jsonToWar(mapJson.units);
 // We can now write the `unitResult.buffer` content to a file named "war3mapUnits.doo" and put it in a .w3x archive!
 ```
 
+
+# TODO: UPDATE README OF TS FORK
```

Comment:
I think with TS support, there should be two examples shown on how to import:
1. The easy TS way (e.g. `import Units from Translator` (or however that syntax is))
2. Importing the built JS (e.g. `new Translator.default().Sounds.jsonToWar(...)`)

At least mention it in the README. I'll be looking at the `examples/` directory and am considering relocating them to the Wiki. If so, I'll take care of that tedious task! 😅

Reply:
Sure I enhance the readme ;)

Reply:
readme updated in #5

Also tested it with: https://stackoverflow.com/questions/8088795/installing-a-local-module-using-npm 
You can try it out yourself. Vs code automatically recognized the "typings" since the link only links to the js files. But the project is ts based! :)

Reply:
Readme with examples updated